### ISSUE TITLE
Failures on TopJoy Falcon

### ISSUE THREAD

**Issue by tolidano** [wimpysworld/umpc-ubuntu#15]
I have just installed the TopJoy Falcon ISO (via the “try” and then “install” option on a USB Stick).
I allocated a 30GB partition for Ubuntu but kept Windows.
After install, it asked me to reboot, which I did.
It did not boot GRUB, but instead Windows. I had to go into the BIOS and swap the UEFI Boot Order so the Ubuntu partition came first.
I rebooted and saw a normal sized (not native resolution) GRUB menu (rotated 90 degrees).
Then, I selected Ubuntu.
The screen is completely garbled (lots of very thin slices). I used basic reasoning to shut it down and start it again.
This time the GRUB menu was at native resolution (the text was much much smaller).
I again selected Ubuntu.
Same issue.
Start over, this time selected recovery mode (always on the 5 kernel)
Now got the pink menu with choices to drop to root or resume.
I selected resume. 
Now I had a native resolution screen, rotated 90 degrees.
I logged in, and attempted to rotate using the system displays menu or the xrandr but both allowed no rotation to occur (and xrandr had an error about gamma detection).
I restarted again and tried normal, same garbled screen for both login and the desktop.

What can I write at the root prompt to provide more info? What can I do to fix it? Should I just try installing again?

**Comment by tolidano** [comment]
Ok, more information:
I reinstalled from scratch but kept WiFi off. It worked fine. Multiple restarts later, all seems ok.
But then I went to software center and updated everything. Upon reboot, it’s back to a garbled screen.

So something has been broken by running a software update. If I could have some info on what might have broken the display, I can attempt to restore it.

Any assistance greatly appreciated.

**Comment by tolidano** [comment]
Ok more info: it installed the 5.0.0-23-generic kernel and made it the default. This kernel does not have the appropriate modifications made by the script here (I’m guessing) and so it fails.

I modified /etc/default/grub to set the GRUB_DEFAULT to “1>2” which picks the 2nd and then 3rd item which should be the valid 4.18 kernel that it installs with.

I tried removing the kernel but dpkg said it wasn’t even there to begin with. This solutions works fine for me, I can boot to windows when necessary but also by default start the 4.18 kernel of Ubuntu which works very well.

Full boot in 20 seconds is pretty stellar.
